.templates/openstack/properties.yml

```diff
@@ -0,0 +1,72 @@
+---
+meta:
+  shield:
+    daemon_public_key: (( vault "secret/dc01/proto/shield/keys/core:public" ))
```

Comment:
This shouldn't go here. This should be defined in a custom  `properties.yml` file once the user has created the environment.